integrations/comet-api.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: integration
+name: Comet API
+description: Use the Comet API for text generation models.
+authors:
+    - name: deepset 
+      socials:
+        github: deepset-ai
+        twitter: deepset_ai
+        linkedin: https://www.linkedin.com/company/deepset-ai/
```

Comment:
You can remove deepset from the authors as this integration is not in `haystack-core-integrations`
```suggestion
```